src/Repositories/CategoryRepository.php

```diff
@@ -24,13 +25,18 @@ public function get(int $categoryId, string $lang, array $with = [])
         $plentyCategoryRepo = pluginApp(PlentyCategoryRepositoryContract::class);
 
         $plentyCategory = $plentyCategoryRepo->get($categoryId, $lang);
+        $categoryDetailsList = $plentyCategory->details;
+        /** @var CategoryDetails $categoryDetails */
+        if($categoryDetailsList instanceof \ArrayAccess) {
+            $categoryDetails = $categoryDetailsList->offsetGet(0);
+        }
 
         /** @var Category $category */
         $category = pluginApp(Category::class);
 
         $category->fillByAttributes([
             'id'       => (int)$plentyCategory->id,
-            'name'     => $plentyCategory->details[0]->name,
+            'name'     => $categoryDetails->name,
             'level'    => (int)$plentyCategory->level - 1,
             'parentId' => (int)$plentyCategory->parentCategoryId,
             'children' => [],
```

Comment:
Da das Repo schon anfangs initialisiert wurde, brauch in Zeile 47 das Repo nicht nochmal geladen werden. Hier lohnt sich dann schon eine globale Variable via Dependency Injection denke ich. Aber bitte vorher mal prüfen, ob das PlentyCategoryRepository auch ordnungsgemäß nach der Suche diverse Filter etc. resettet.